app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
@maxdumas I've been thinking about `automatic` vs. `independent` (or something else) and am on the fence. Out of curiosity, what is it about `automatic` that you prefer?

(Re `internal`: I'm not feeling it, since an internal property, to me, would be something that the client can't even read (à la typical encapsulation), which isn't the case here.)